subsys/bluetooth/host/hci_core.h

```diff
@@ -209,3 +209,5 @@ int bt_le_adv_start_internal(const struct bt_le_adv_param *param,
 			     const struct bt_data *sd, size_t sd_len,
 			     const bt_addr_le_t *peer);
 void bt_le_adv_resume(void);
+
+int bt_set_rpa_timeout(u16_t timeout);
```

Comment:
Do you intend this to be a public or internal API? If it is to be public then it needs to go to `include/bluetooth/` since `hci_core.h` is an internal header file. Where do you expect this to be called.

In general your commit message could use a lot more "flesh on the bones", in particular answering the most important question any commit message should answer, i.e. "why?" (the answer to "what?" can be found simply by looking at the diff, so it's less valuable, but still ok to have in the commit message).

Reply:
> Do you intend this to be a public or internal API? 

I intended it to be a public interface because I think it would be a useful addition. however, as this is a specific need for our product, I could easily accept making it vendor specific. Moved prototype to bluetooth.h.